c/pthread-ext/06_ticket_true-unreach-call.c

```diff
@@ -1,5 +1,8 @@
 extern void __VERIFIER_assume(int);
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+int __global_lock;
+void __VERIFIER_atomic_begin() { __VERIFIER_assume(__global_lock==0); __global_lock=1; return; }
```

Comment:
Monty, `__VERIFIER_atomic_begin` and `__VERIFIER_atomic_end` should be just declarations. Their meaning is defined by the SVCOMP standard, the same as e.g. `__VERIFIER_error`, `__VERIFIER_assume`, etc.